### PR TITLE
Document ZSH sublime plugin

### PR DESCRIPTION
**Pull request by GuGuss**
https://github.com/robbyrussell/oh-my-zsh/wiki/Plugins#sublime

### CODE REVIEW

File: SublimeText/README.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ Go ahead and [download](http://www.sublimetext.com/) it. Open the **.dmg** file,
 
 Sublime Text is not free, but it has an unlimited "evaluation period". Anyhow, we're going to be using it so much that even the seemingly expensive $60 price tag is worth every penny. If you can afford it, It is adviced that you [support](http://www.sublimetext.com/buy) this awesome tool. :)
 
-Let's create a shortcut so we can launch Sublime Text from the command-line:
+ZSH has a plugin to create shortcuts to launch Sublime Text from the command-line. Simply open your `~/.zshrc` file and make sure that the  `sublime` plugin is part of your plugins:
```
Comment (sb2nov) [inline SublimeText/README.md]: I think this is oh-my-zsh specific as prezto doesn't seem to have this plugin based on the comments in the PR https://github.com/sorin-ionescu/prezto/pull/671. 

Let's improve the documentation on creating the shortcut instead as that is shell agnostic as well. Thoughts?

Comment (GuGuss) [inline SublimeText/README.md]: Agreed. I'm a oh-my-zsh fan ;)

Comment (sb2nov) [inline SublimeText/README.md]: Thanks, I'll merge the PR once we update the documentation 👍

**Comment by simeg** [comment]
I vote for closing this due to inactivity.

Yes/no?

@sb2nov @Kyslik

**Comment by sb2nov** [comment]
+1 to closing as outdated. @GuGuss feel free to reopen whenever you want to work on this again.

**Comment by Kyslik** [comment]
Interesting I don't use sublime package (for ZSH) and I have the `subl` command :D (and not aliased).

I bet I cmd+c & cmd+v some command somewhere.

**Comment by sb2nov** [comment]
Might be based on how you install it. The Sublime documentation has the command to create a symlink https://github.com/sb2nov/mac-setup/blob/master/SublimeText/README.md#use-cli-to-open-file

**Comment by Kyslik** [comment]
Ye, I used cask.

https://github.com/caskroom/homebrew-cask/blob/master/Casks/sublime-text.rb

**Pull request closed without merging**